docs/docs/components/calendar-input.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Calendar Input
+---
+
+import { Demo } from '@site/src/components/DemoComponent.jsx'
```

Comment:
I think these imports should go in the example? Also, it looks like the example is missing code block formatting?

Reply:
Makes sense now (after you explained it to me more 😅)